internal/backend/azure/config.go

```diff
@@ -71,4 +72,7 @@ func (cfg *Config) ApplyEnvironment(prefix string) {
 	if cfg.AccountSAS.String() == "" {
 		cfg.AccountSAS = options.NewSecretString(os.Getenv(prefix + "AZURE_ACCOUNT_SAS"))
 	}
+	if cfg.Domain == "" {
+		cfg.Domain = os.Getenv("AZURE_DOMAIN")
```

Comment:
The environment variable should be included in the list in `doc/040_backup.rst`.